app/src/main/java/org/wikipedia/editactionfeed/AddTitleDescriptionsFragment.kt

```diff
@@ -51,7 +50,7 @@ class AddTitleDescriptionsFragment : Fragment() {
     private var languageCodesToList: MutableList<String> = arrayListOf()
     var langFromCode: String = app.language().appLanguageCode
     var langToCode: String = if (app.language().appLanguageCodes.size == 1) "" else app.language().appLanguageCodes[1]
-    var source: Int = EDIT_TASKS_TITLE_DESC_SOURCE
+    var source: Int = InvokeSource.EDIT_FEED_TITLE_DESC.ordinal
```

Comment:
The type should be `InvokeSource` instead of Int.

Reply:
Done